### PR TITLE
feat(#9): home endpoints

### PR DESCRIPTION
**Pull request by h1alexbel**
@l3r8yJ take a look, please

ref #9

---

## PR-Codex overview
The focus of this PR is to add new dependencies, implement routes for home endpoints, and create error handling structures.

### Detailed summary
- Added `serde_json` dependency
- Added `axum` and `tokio` dependencies
- Implemented routes for home endpoints
- Created error handling structures for route errors

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by l3r8yJ — CHANGES_REQUESTED** [review]
@h1alexbel i think that it would be better to move `home.rs` and `rs_err.rs` to different folders and include them from modules

File: clippy.toml [hunk 1]
```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-disallowed-names = ["not-jeff"]
\ No newline at end of file
+disallowed-names = ["not-jeff"]
```
[the remark this answers is not in the document]
Comment (h1alexbel) [reply clippy.toml]: @l3r8yJ fixed

**Review by l3r8yJ — APPROVED** [review]
@h1alexbel looks good to me!

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by l3r8yJ** [comment]
blocked by #11

**Comment by h1alexbel** [comment]
@l3r8yJ can we try to install required `rustc` in `install` section inside `.rultor.yml`?

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@rultor stop

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by l3r8yJ** [comment]
@h1alexbel looks like we need to debug rultor...

**Comment by h1alexbel** [comment]
@l3r8yJ it's in outage for now

**Comment by l3r8yJ** [comment]
@h1alexbel you can close this pr, remove remote branch and reopen pr, maybe it will help

**Comment by h1alexbel** [comment]
@l3r8yJ this repository is not connected to this outage, we can only wait for now

**Comment by l3r8yJ** [comment]
@h1alexbel whatsupp here?

**Comment by h1alexbel** [comment]
@l3r8yJ `@rultor` still in the outage. we can merge this pr manually

**Comment by l3r8yJ** [comment]
@h1alexbel sure, wont wait

**Pull request merged**